display_information/display-grid/src/main/java/com/esri/samples/display_grid/DisplayGridController.java

```diff
@@ -58,9 +59,17 @@
   }
 
   public void initialize() {
-    ArcGISMap map = new ArcGISMap(Basemap.createImagery());
+
+    // authentication with an API key or named user is required to access basemaps and other location services
+    String yourAPIKey = System.getProperty("apiKey");
+    ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+    // create a map and set its initial viewpoint
+    ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_IMAGERY_STANDARD);
```

Comment:
was there a reason this one didn't get updated to `mapView.setViewpoint`? just curious.

Reply:
@Rachael-E originally I only changed this where the `Viewpoint` had been set in the map constructor. So this had always had the `Viewpoint` set on the map separate to the constructor. However since we're going for it now, I'll refactor and test in any Samples that have had the key added / already been edited in some way.